osu.Game/Screens/OnlinePlay/Playlists/PlaylistsLoungeSubScreen.cs

```diff
@@ -16,7 +20,38 @@ public class PlaylistsLoungeSubScreen : LoungeSubScreen
         [Resolved]
         private IAPIProvider api { get; set; }
 
-        protected override FilterControl CreateFilterControl() => new PlaylistsFilterControl();
+        private Dropdown<PlaylistsCategory> categoryDropdown;
+
+        protected override IEnumerable<Drawable> CreateFilterControls()
+        {
+            categoryDropdown = new SlimEnumDropdown<PlaylistsCategory>
+            {
+                RelativeSizeAxes = Axes.None,
+                Width = 160,
+            };
+
+            categoryDropdown.Current.BindValueChanged(_ => UpdateFilter());
+
+            return base.CreateFilterControls().Append(categoryDropdown);
+        }
+
+        protected override FilterCriteria CreateFilterCriteria()
+        {
+            var criteria = base.CreateFilterCriteria();
+
+            switch (categoryDropdown.Current.Value)
+            {
+                case PlaylistsCategory.Normal:
+                    criteria.Category = "normal";
```

Comment:
Could as well make these raw strings if they're moving anyways.

Reply:
What do you mean by "raw"? Do you mean verbatim?

Reply:
Right, yeah, that is what I tried to say.